fury/ui/elements.py

```diff
@@ -3052,3 +3055,94 @@ def directory_click_callback(self, i_ren, _obj, listboxitem):
                 self.set_slot_colors()
         i_ren.force_render()
         i_ren.event.abort()
+
+
+class Visualizer(UI):
+    def __init__(self, position):
+        super(Visualizer, self).__init__(position)
+        self.actor_list = []
+
+    def _setup(self):
+        """Setup this UI component.
+
+        Create a Canvas(Panel2D).
+        """
+        self.canvas = Panel2D((400, 400))
+        self.canvas.background.on_left_mouse_button_pressed = self.left_button_pressed
+        self.canvas.background.on_left_mouse_button_dragged = self.left_button_dragged
+
```

Comment:
I was planning to add two new `Panel2D`,
One at the bottom to select the mode (ie. Line, Polygon, Circle, Freehand) and
Another at left or right side which would help us to modify the properties of the current selected elements.